### ISSUE TITLE
Remove ability to upload folders into the file upload area

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/happy-heron#2842]
As part of hierarchical file support, we are moving to a model where users upload a single ZIP, which is then expanded on the server to provide access to the internal structure.  So we do not want to allow users to upload folders in H2.

**Comment by justinlittman** [comment]
Why not? The current functionality allows a user to quickly add a set of files. How does this conflict with the proposed new functionality?

**Comment by peetucket** [comment]
We discussed and have the following reasons:

- unless we can show the hierarchical structure in the drag/drop file upload area this same as is shown in #2843, we will end up having two separate views of incoming hierarchy, which may be confusing
- related: files uploaded in the drag/drop area allow for descriptions/individual deletions, which is different than what will happen in #2843 for the zip upload
- we could end up with potential filename clashes if two folders are uploaded separately
- potential browser incompatibilities?

Basically, for the various reasons we decided to move to a single ZIP upload...

**Comment by justinlittman** [comment]
Currently, a user can drag-and-drop a folder and the files will be extracted. We don't have the ability to stop a user from dragging-and-dropping. How do want it to behave?

**Comment by peetucket** [comment]
I thought we added something to allow dropping folders, didn't realize it just worked by default.  Perhaps there is nothing we can/should do then.  We will now prevent dupe filenames via #2800, so it may just work fine if they happen to drop a folder?  @amyehodge can help decide

**Comment by mjgiarlo** [comment]
AFAICT, @justinlittman is correct that users are free to drag-and-drop folders and have always been able to. When they do so, no path information is recorded; just the filenames.

We could look into beginning to **prevent** this, but I'm not sure that's what's been asked for. What do you think, @amyehodge ?

**Comment by amyehodge** [comment]
I'm gonna defer on answering this until after some discussions today about the workflows for everything. I'm hoping we can simplify some things.

**Comment by mjgiarlo** [comment]
@amyehodge Any new thoughts on this or should we pause a while and revisit later?

**Comment by amyehodge** [comment]
@mjgiarlo I'd like to wait until we discuss the new mockups with the team, but I'm thinking we will likely not do this.

**Comment by mjgiarlo** [comment]
Sounds good. :)

**Comment by peetucket** [comment]
After talking to Justin L this morning, this ticket may be a no-op.  The current behavior allows users to drag a folder into that area and it will simply flatten the list of files present if you do this.  This would match the expected behavior for that upload  mechanism.

**Comment by mjgiarlo** [comment]
That sounds right to me.

**Comment by mjgiarlo** [comment]
@peetucket @amyehodge Not to pester too much... shall we close this issue?

**Comment by peetucket** [comment]
I believe we can close this.
